tests/unit/algo/visual_prompting/backbones/test_tiny_vit.py

```diff
@@ -20,7 +20,7 @@
 
 
 class TestConv2d_BN:  # noqa: N801
-    def setup(self) -> None:
+    def setup_method(self) -> None:
```

Comment:
@sungchul2 
Please update them with the common practice such as
```python
@pytest.fixture()
def conv2d_bn(self):
    return Conv2d_BN(a=1, b=1)
```

Reply:
I'll update it at https://github.com/openvinotoolkit/training_extensions/pull/2834.